Gemfile

```diff
@@ -54,21 +57,20 @@ gem 'possibly', '~> 0.2.0'
 # has been fixed for 4.0.4 https://github.com/collectiveidea/delayed_job/issues/697
 # However, some tests break for 4.0.4, so keeping 4.0.2 for now
 gem 'delayed_job', "~> 4.0.0", "<= 4.0.2"
-gem 'delayed_job_active_record' # , "~> 4.0.0"
-gem 'json', "~>1.8.0"
-gem 'multi_json', "~>1.7.3" # 1.8.0 caused "invalid byte sequence in UTF-8" at heroku
-gem 'web_translate_it'
-gem 'rails-i18n'
+gem 'delayed_job_active_record', "4.0.3"
+
+gem 'json', "~>1.8.3"
+gem 'multi_json', "~>1.7.9" # 1.8.0 caused "invalid byte sequence in UTF-8" at heroku
+gem 'web_translate_it', "~>2.1.8"
+gem 'rails-i18n', '~> 4.0', '>= 4.0.8'
```

Comment:
That's a good point. I'll change that.

In general, if all gems would follow strictly Semantic Versioning, we should be some to limit the updates to MINOR versions, since only MAJOR versions add breaking changes. However, according to my tests, that's not the case. Also MINOR updates will break. So better to be on the safe side and be a bit more pessimistic.